pages/Product/components/Content/index.jsx

```diff
@@ -1,27 +1,34 @@
-import React, { Component } from 'react';
+import React, { Component, Fragment } from 'react';
 import PropTypes from 'prop-types';
-// Import Portal from '@shopgate/pwa-common/components/Portal';
-// Import * as portals from '@shopgate/pwa-common-commerce/product/constants/Portals';
+import Portal from '@shopgate/pwa-common/components/Portal';
+import * as portals from '@shopgate/pwa-common-commerce/product/constants/Portals';
 // Import Reviews from 'Components/Reviews';
 // Import TaxDisclaimer from '@shopgate/pwa-ui-shared/TaxDisclaimer';
 // Import ImageSlider from './components/ImageSlider';
-// Import Header from './components/Header';
+import Header from '../Header';
 // Import VariantSelects from './components/VariantSelects';
 // Import Options from './components/Options';
 // Import Description from './components/Description';
 // Import Properties from './components/Properties';
 import connect from './connector';
+import ProductContext from '../../context';
 
 /**
  * The product content component.
  */
 class ProductContent extends Component {
   static propTypes = {
-    name: PropTypes.string,
+    baseProductId: PropTypes.string,
+    id: PropTypes.string,
```

Comment:
We use `productId` everywhere. Let's keep it consistent and rename this to `productId`, please.

Reply:
Changed